project-templates/import-records/Makefile

```diff
@@ -0,0 +1,102 @@
+
```

Comment:
Aside: I wish we could standardise on this stuff. Some places we use make files, some places we use grunt.